tests/solvers/test_aeossp_standard_mwis.py

```
@@ -311,6 +323,150 @@ def __init__(self, *args, **kwargs):
+    class FakeProcessPoolExecutor:
+        def __init__(self, *args, **kwargs):
+            pass
+
+        def __enter__(self):
+            return self
+
+        def __exit__(self, *args):
+            return False
+
+        def map(self, function, *iterables):
+            return [function(*args) for args in zip(*iterables)]
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)